PWGCF/EbyEFluctuations/Tasks/RobustFluctuationObservables.cxx

```diff
@@ -1734,172 +1786,177 @@ struct RobustFluctuationObservables {
     } // end of track loop
 
     // ##### v0 analysis
-    for (const auto& v0 : v0s) {
-      double v0rad = v0.v0radius();
-      histosK0S.fill(HIST("hK0Sradius"), v0rad);
+    if (flagIncludeQAHistK0S) {
 
-      // Cut on dynamic columns
-      if (v0.v0cosPA() < 0.99) // v0setting_cospa)
-        continue;
+      for (const auto& v0 : v0s) {
+        double v0rad = v0.v0radius();
+        histosK0S.fill(HIST("hK0Sradius"), v0rad);
 
-      histosK0S.fill(HIST("hMassK0Short"), v0.mK0Short());
+        // Cut on dynamic columns
+        if (v0.v0cosPA() < 0.99) // v0setting_cospa)
+          continue;
 
-      if (v0rad < 0.8 || v0rad > 20) // v0setting_radius)
-        continue;
+        histosK0S.fill(HIST("hMassK0Short"), v0.mK0Short());
 
-      // if (v0rad < v0RadiusMin || v0rad > v0RadiusMax || v0.eta() > assocEtaMax || v0.eta() < assocEtaMin || v0.v0cosPA() < v0Cospa) {
-      if (fabs(v0.eta()) > 0.75)
-        continue;
+        if (v0rad < 0.8 || v0rad > 20) // v0setting_radius)
+          continue;
 
-      const auto& posDaughterTrack = v0.posTrack_as<DaughterTracks>();
-      const auto& negDaughterTrack = v0.negTrack_as<DaughterTracks>();
+        // if (v0rad < v0RadiusMin || v0rad > v0RadiusMax || v0.eta() > assocEtaMax || v0.eta() < assocEtaMin || v0.v0cosPA() < v0Cospa) {
+        if (fabs(v0.eta()) > 0.75)
+          continue;
 
-      double etaDauPos = posDaughterTrack.eta();
-      double etaDauNeg = negDaughterTrack.eta();
+        const auto& posDaughterTrack = v0.posTrack_as<DaughterTracks>();
+        const auto& negDaughterTrack = v0.negTrack_as<DaughterTracks>();
```

Comment:
I would let the compiler optimizations to decide if an intermediate storage is needed, i.e. use directly the right hand without defining a specific variable for it which will induce the compiler to create an specific storage for it which perhaps it is not needed (it is a suggestion)